distributionviewer/core/static/js/app/components/views/chart.js

```diff
@@ -15,20 +18,23 @@ function formatData(item) {
     result.push({
       x: data[i]['refRank'] || data[i]['b'],
       y: data[i]['c'] * 100,
+      p: data[i]['p'] * 100,
```

Comment:
`p` is the proportion that the bucket makes up the whole. Can I see the chart somehow?